### PR TITLE
Use files from php-tools repo.

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,13 +30,16 @@ env:
...
-    - PATH=${HOME}/gcloud/google-cloud-sdk/bin:${PATH}
+    - PATH=${HOME}/google-cloud-sdk/bin:${PATH}
     - PHP_CGI_PATH=/home/travis/.phpenv/shims/php-cgi
     - TEST_BUILD_DIR=$TRAVIS_BUILD_DIR
 
 before_install:
-  - php dump_credentials.php
-  - travis_retry testing/install_test_deps.sh
+  - pushd ${HOME}
+  - git clone https://github.com/GoogleCloudPlatform/php-tools.git
```
Comment (bshaffer) [inline .travis.yml]: Why are we cloning this instead of just adding it to `require-dev` in `composer.json`?

Comment (tmatsuo) [inline .travis.yml]: I can do that, but currently we don't have top level `composer.json` and I just thought it's not warrant to add the top level `composer.json`. Each test has it in their `require-dev`.

Comment (bshaffer) [inline .travis.yml]: Ahh, I see. Yeah let's not worry about it then.

**Comment by bshaffer** [comment]
LGTM

**Comment by tmatsuo** [comment]
Thanks!

**Pull request merged**